cluster/vagrant-kubernetes/setup_master.sh

```diff
@@ -30,7 +30,15 @@ systemctl enable cockpit.socket && systemctl start cockpit.socket
 rm -rf /var/lib/kubelet
 
 # Create the master
-kubeadm init --pod-network-cidr=10.244.0.0/16 --token abcdef.1234567890123456
+cat >/etc/kubernetes/kubeadm.conf <<EOF
```

Comment:
is this change still required, or is it essentially a no-op now that the admission control feature isn't being used?

Reply:
If nothing at all has changed with respect to pod networking, then it's a no-op. @rmohr's suggested change implied something was afoot with podNetwork so I left it.